charts/lagoon-build-deploy/ci/linter-values.yaml

```diff
@@ -5,3 +5,10 @@ lagoonTargetName: ci-local-control-k8s
 taskSSHHost: lagoon-core-ssh.lagoon.svc
 taskSSHPort: 2020
 taskAPIHost: http://lagoon-core-api.lagoon.svc:80
+extraArgs:
```

Comment:
probably dont need to define these here really, the linter test doesnt care if harbor is enabled or not

Reply:
I've only added them here in case people look to the linter for a template :wink: